src/fu-util.c

```diff
@@ -3650,7 +3650,7 @@ fu_util_check_daemon_version(FuUtilPrivate *priv, GError **error)
 static gboolean
 fu_util_check_polkit_actions(GError **error)
 {
-#ifdef HAVE_POLKIT
+#ifdef VERIFY_POLKIT_ACTIONS
```

Comment:
I think we need to have PolicyKit working -- this allows any user to do anything.

Reply:
It is intended to work. But the check of the policy file does not work when strictly confined.  Snapd is responsible of installing the policy with the fwupd interface. The path would be `/var/lib/snapd/hostfs/usr/share/polkit-1/actions/snap.fwupd.interface.polkit.org.freedesktop.fwupd.policy` and we would need to give permission to read that file.

Reply:
By default can the confined `fwupdmgr` not access that file then?  

I follow the intent of the new config option, but any other way we can dynamically verify that we don't need to check for it?  Perhaps can we check that we're in a confined snap environnment in some way?  environment variable?

Reply:
Yes, we can use an environment variable. We can define it in the wrapper.

Reply:
OK, please make a new environment variable and document it in in `docs/env.md`.  I don't think it needs to be snap specific.  Maybe `FWUPD_IGNORE_POLKIT`?

Reply:
I am afraid `FWUPD_IGNORE_POLKIT` is not really the right thing. We do not ignore polkit, we just want to not check if the policies were installed. Would `FWUPD_NOCHECK_POLKIT` be ok?

Reply:
I have pushed changes using `FWUPD_POLKIT_NOCHECK`

Reply:
Could we rename FU_PATH_KIND_POLKIT_ACTIONS to FU_PATH_KIND_POLKIT_POLICY and return the file path? Otherwise there's not much point in FU_PATH_KIND_POLKIT_ACTIONS existing...